src/Elmish.WPF/BindingInternal.fs

```diff
@@ -4,9 +4,18 @@ open System.Collections.ObjectModel
 open System.Windows
 
 open Elmish
+open Microsoft.Extensions.Logging
 
 
 
+type ViewModelHelper = {
```

Comment:
This is a good idea.  Let's improve this and make it into its own PR that gets merged first.

Use this formatting and naming...
```fsharp
type UnchangingViewModelArgs =
  { performanceLogThresholdMs: int
    log: ILogger
    logPerformance: ILogger }

type ViewModelArgs<'model, 'msg> =
  { model: 'model
    dispatch: 'msg -> unit
    nameChain: string
    unchangingArgs: UnchangingViewModelArgs }
```

...and make use of those types (though feel free to suggest better names).

Reply:
I need `nameChain` to not be included as a first class argument to the Create function. It's only there for logging purposes. I'd prefer to have them grouped by purpose, not mutability.

Reply:
See #451

Reply:
Ok.  Your `map` function minimizing the pain of including `nameChain` with the other logging arguments.